metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_config.py

```diff
@@ -4,22 +4,11 @@
 from pydantic import Field, root_validator
 
 from datahub.configuration.common import AllowDenyPattern
-from datahub.ingestion.source.sql.sql_common import SQLAlchemyStatefulIngestionConfig
 from datahub.ingestion.source_config.sql.snowflake import (
     SnowflakeConfig,
     SnowflakeProvisionRoleConfig,
 )
-from datahub.ingestion.source_config.usage.snowflake_usage import (
-    SnowflakeStatefulIngestionConfig,
-    SnowflakeUsageConfig,
-)
-
-
-class SnowflakeV2StatefulIngestionConfig(
```

Comment:
@mayurinehate : can you confirm that this is safe to remove.

Reply:
yes, it is safe to remove.